### PR TITLE
Added prison to categories.txt

### LINKED ISSUE #9121: POI prison doesn't show up in search results
⚠ Have you searched for similar, already existing issues?

Yes

**Describe the issue**

When searching for the POI type prison it only shows places which have prison in the name not the POI. 

**Steps to reproduce**
1. Go to search
2. Search for prision
3. See that there are only results with 'prison' in the search results

**Expected behaviour**
I would expect it to show all prisons including the POI 

**System information:**
 - Operating system and its version: Android 14, OM Version 2024.08.16-5

### PR DESCRIPTION
**Pull request by ProgramminCat**
Resolves #9121

### CODE REVIEW

**Review by vng — APPROVED** [review]
Can't imagine who cares about the "prison" search category, but technically PR is ok ..

**Comment by vng** [comment]
Any volunteers to translate?

File: data/categories.txt [hunk 1]
```diff
@@ -16694,3 +16694,8 @@ uk:Центр мистецтв
 vi:Trung tâm nghệ thuật
 zh-Hans:艺术中心
 zh-Hant:藝術中心
+
+amenity-prison
```
Comment (andergd) [inline data/categories.txt:16698]: Cannot PR but here it is in Spanish:

`es:cárcel`

And I don't know if will be used but also in Basque:

`eu:espetxe`

**Review by andergd — CHANGES_REQUESTED** [review]

File: data/categories.txt (hunk 1 above)
Comment (biodranik) [reply data/categories.txt:16698]: en:prison
ar:السجن
be:турма
bg:затвор
ca:presó
cs:vězení
da:Fængsel
de:Gefängnis
el:φυλακή
es:cárcel|prisión
et:vangla
eu:espetxe|kartzela
fa:زندان
fi:vankila
fr:prison
he:כֵּלֵא
hi:कारागार
hu:börtön
id:penjara
it:prigione
ja:刑務所
ko:감옥
mr:तुरुंग
nb:fengsel
nl:gevangenis
pl:więzienie
pt:prisão
ro:închisoare
ru:тюрьма
sk:väzenie
sv:fängelse
sw:jela
th:คุก
tr:hapishane
uk:в'язниця
vi:nhà tù
zh-Hans:监狱
zh-Hant:監獄

Comment (NeatNit) [reply data/categories.txt:16698]: Hebrew (he): בית כלא

Comment (biodranik) [reply data/categories.txt:16698]: @NeatNit what's the difference with the autotranslation?

Comment (NeatNit) [reply data/categories.txt:16698]: > @NeatNit what's the difference with the autotranslation?

Removal of Nikkud for one thing (the tiny dots and symbols around the letters), but the bigger difference is, my suggestion is the full term for prison (loosely "house of imprisonment") whereas the auto-translation is just the last word. It *can* be used on its own to mean pretty much the same thing but in my subjective opinion, my suggestion is the clearer term.

If you insist on the one-word name, then at least remove Nikkud: כלא

Comment (biodranik) [reply data/categories.txt:16698]: This is for search, so two or more words/synonyms should be added (what users likely type in the search input box). Can you please write the final he:...|... string with all synonyms?

**Review by muralito — CHANGES_REQUESTED** [review]
Please add this spanish search terms for "prison" 

es:prisión|cárcel|penal|penitenciaría|presidio|correcional|centro penitenciario|centro de reclusión

**Comment by pastk** [comment]
@ProgramminCat if you feel inspired please also add search terms for Arts and Crafts shops (`shop-craft`) - better in a separate PR.

**Comment by pastk** [comment]
A sidenote - do we have a way to list all POI types that don't have search terms yet? Similar to https://github.com/organicmaps/organicmaps/pull/6796

File: data/categories.txt (hunk 1 above)
Comment (jimcarst) [reply data/categories.txt:16698]: nl is ok. There are too many synonyms and only this one is suitable for search.

File: data/categories.txt [hunk 2]
```diff
@@ -16694,3 +16694,9 @@ uk:Центр мистецтв
...
+en:prison
+fr:prison
+ru:тюрьма
+es:prisión|cárcel|penal|penitenciaría|presidio|correcional|centro penitenciario|centro de reclusión
```
[the remark this answers is not in the document]
Comment (patepelo) [reply data/categories.txt:16702]: Totally agree, wantes to comment on this. This undermines search performance and accuracy.

Plese keep only: prisión|cárcel

[the remark this answers is not in the document]
Comment (muralito) [reply data/categories.txt]: Just remove them if it hurts performance, but the Spanish language is just like it is, with lots of variants and synonyms. I don't know statistics for the everyday use of those terms to choose which ones suits best for the use case. In worst case, the user would try searching for several of them until he gets the results.

**Comment by soshial** [comment]
lv:cietums

File: data/categories.txt (hunk 1 above)
Comment (NeatNit) [reply data/categories.txt:16698]: > This is for search, so two or more words/synonyms should be added (what users likely type in the search input box). Can you please write the final he:...|... string with all synonyms?

Sorry, I missed this message until now. 

he:כלא|בית כלא|בית סוהר

**Comment by ProgramminCat** [comment]
> @ProgramminCat if you feel inspired please also add search terms for Arts and Crafts shops (`shop-craft`) - better in a separate PR.

https://github.com/organicmaps/organicmaps/pull/9492

File: data/categories.txt [hunk 3]
```diff
@@ -16694,3 +16694,44 @@ uk:Центр мистецтв
...
+ca:presó
+cs:vězení
+da:Fængsel
+de:Gefängnis
+el:φυλακή
+et:vangla
+eu:espetxe|kartzela
+fa:زندان
+fi:vankila
+fr:prison
```
Comment (Arthur-GYT) [inline data/categories.txt:16715]: LGTM

**Review by Arthur-GYT — APPROVED** [review]

**Comment by NeatNit** [comment]
Hebrew is out of order - the languages should be sorted alphabetically by language code. Otherwise Hebrew is good.

File: data/categories.txt [hunk 4]
```diff
@@ -16694,3 +16694,43 @@ uk:Центр мистецтв
...
+pt:prisão
+ro:închisoare
+sk:väzenie
+sv:fängelse
+sw:jela
+th:คุก
+tr:hapishane
+uk:в'язниця
+vi:nhà tù
+zh-Hans:监狱
```
Comment (pastk) [inline data/categories.txt:16736]:
```suggestion
en:prison
ar:السجن
be:турма
bg:затвор
ca:presó
cs:vězení
da:Fængsel
de:Gefängnis
el:φυλακή
es:prisión|cárcel
et:vangla
eu:espetxe|kartzela
fa:زندان
fi:vankila
fr:prison
he:כלא|בית כלא|בית סוהר
hi:कारागार
hu:börtön
id:penjara
it:prigione
ja:刑務所
ko:감옥
lv:cietums
mr:तुरुंग
nb:fengsel
nl:gevangenis
pl:więzienie
pt:prisão
ro:închisoare
ru:тюрьма
sk:väzenie
sv:fängelse
sw:jela
th:คุก
tr:hapishane
uk:в'язниця
vi:nhà tù
zh-Hans:监狱
```
Re-sorted all lines

**Review by pastk — APPROVED** [review]

**Comment by pastk** [comment]
@ProgramminCat please remove Latvian `lv:cietums` for now - it causes tests to fail (Latvian should be enabled properly elsewhere, it'll be done in #9486)

File: data/categories.txt [hunk 5]
```diff
@@ -16694,3 +16694,42 @@ uk:Центр мистецтв
 vi:Trung tâm nghệ thuật
 zh-Hans:艺术中心
 zh-Hant:藝術中心
+
+amenity-prison
+en:prison
+ar:السجن
```
Comment (Colonel-Dixie) [inline data/categories.txt:16700]: ar:السجن needs improvement, it translate to "the prison"    
The correct one should be ar:سجن which is just " prison.

Comment (pastk) [reply data/categories.txt:16700]: Thanks!
https://github.com/organicmaps/organicmaps/pull/9555

**Pull request merged**